apps/authentication-portal/src/main/webapp/extensions/branding-preferences.jsp

```diff
@@ -1,21 +1,21 @@
-<%--
-  ~ Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com) All Rights Reserved.
-  ~
-  ~ WSO2 Inc. licenses this file to you under the Apache License,
-  ~ Version 2.0 (the "License"); you may not use this file except
-  ~ in compliance with the License.
-  ~ You may obtain a copy of the License at
-  ~
-  ~ http://www.apache.org/licenses/LICENSE-2.0
-  ~
-  ~ Unless required by applicable law or agreed to in writing,
-  ~ software distributed under the License is distributed on an
-  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-  ~ KIND, either express or implied.  See the License for the
-  ~ specific language governing permissions and limitations
-  ~ under the License.
---%>
+<!--
+~ Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
+~
+~ This software is the property of WSO2 LLC. and its suppliers, if any.
+~ Dissemination of any information or reproduction of any material contained
+~ herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+~ You may not alter or remove any copyright or other notice from copies of this content."
+-->
+
+<%@ page import="org.apache.commons.lang.StringUtils" %>
 
 <%
     String supportEmail = "";
+    String privacyPolicyURL = "";
+    String cookiePolicyURL = "";
+
+    if (StringUtils.equals(request.getParameter("sp"), "Console")) {
```

Comment:
Don't we have module level constants for "Console" ?

Reply:
We have a set of constants in [1], but they are not used in any of the portals. I will track a product is issue for this.

[1] https://github.com/wso2/identity-apps/blob/master/components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalConstants.java#L43